### PR TITLE
Execution progress

### PR DESCRIPTION
**Pull request by dczulada**
During validation phase
Screen Shot 2021-05-06 at 4 01 37 PM

During calculation phase
Screen Shot 2021-05-06 at 3 52 34 PM

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: app/models/test_execution.rb [hunk 1]
```diff
@@ -29,14 +29,17 @@ def build_document(file)
   # I dont think this belongs here and it will need to eventually be moved to a
   # more approperiate location
   def validate_artifact(validators, artifact, options = {})
+    total_files = artifact.count
+    validated_files = 0
+    validation_tracker = tracker
```
Comment (lmd59) [inline app/models/test_execution.rb:34]: Is there a value to renaming this?

**Review by lmd59 — COMMENTED** [review]
Are there any tests worth adding for this? Might be able to add a quick cucumber check to an existing scenario or if not that, a unit test.

File: test/models/cms_program_task_test.rb [hunk 2]
```diff
@@ -169,6 +169,22 @@ def test_telehealth_calcuations
     end
   end
 
+  def test_calcuation_status
```
Comment (lmd59) [inline test/models/cms_program_task_test.rb:172]:
```suggestion
  def test_calculation_status
```

**Review by lmd59 — APPROVED** [review]
